packages/venia-ui/lib/components/App/app.js

```diff
@@ -96,7 +95,6 @@ const App = props => {
             </Main>
             <Mask isActive={hasOverlay} dismiss={handleCloseDrawer} />
             <Navigation />
-            <MiniCart />
```

Comment:
Should remain, just be deprecated, for now.

Reply:
Added back in https://github.com/magento/pwa-studio/pull/2494/commits/6524bd78dd6a6836ac1d3684ae6d9e5180d3265f and added a deprecation notice in https://github.com/magento/pwa-studio/pull/2494/commits/99755696b0dd3c8426b919ed282e505fd801b32c.